daml-foundations/daml-tools/da-hs-daml-cli/DA/Cli/Damlc.hs

```diff
@@ -602,56 +604,58 @@ execInspectDar inFile = do
 execMigrate ::
        Compiler.Options -> FilePath -> FilePath -> Maybe FilePath -> Command
 execMigrate opts inFile1 inFile2 mbDir = do
```

Comment:
I suspect we need a Migrate module that has most of this function in.

Reply:
yes, I'll do a separate PR for reorganizing the code once the functionality is complete.